README.rst

```diff
@@ -129,11 +129,10 @@ Advised linters alongside pylint
 --------------------------------
 
 Projects that you might want to use alongside pylint include ruff_ (**really** fast,
-with builtin auto-fix and a growing number of checks taken from popular
-linters but implemented in ``rust``) or flake8_ (faster and simpler checks with very few false positives),
-mypy_, pyright_ or pyre_ (typing checks), bandit_ (security oriented checks), black_ and
-isort_ (auto-formatting), autoflake_ (automated removal of unused imports or variables),
-pyupgrade_ (automated upgrade to newer python syntax) and pydocstringformatter_ (automated pep257).
+with builtin auto-fix and a large number of checks taken from popular linters, but
+implemented in ``rust``) or flake8_ (a framework to implement your own checks in python using ``ast`` directly),
```

Comment:
`ruff` is a (originally) a rewrite of `flake8` in rust. If we remove `pyupgrade` we might as well remove that as well, although I think it would be good to keep `pyupgrade` in there. It influenced `ruff` directly and is still being maintained, feels like it deserves some credit.

Reply:
flake8 is still useful if you need to create your own plugin with ``ast`` without re implementing the noqa /file handling or using astroid, Let's add pyupgrade back then.